src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
 // write your code here
+const collection = document.querySelectorAll('.population');
+
+const population = [...collection]
+  .map((item) => item.innerText.split(',').join(''))
+  .map((item) => Number(item))
+  .filter((item) => !isNaN(item)); // Переконуємося, що це число
```

Comment:
- `map` doesn't mutate an array, so you don't need to create a copy
- You can prepare one `map`
- Don't leave comments, especially in other language except for English, and this comment looks like ChatGPT leaves it

Reply:
> * `map` doesn't mutate an array, so you don't need to create a copy
> * You can prepare one `map`
> * Don't leave comments, especially in other language except for English, and this comment looks like ChatGPT leaves it

that map is not for a reason of creating copy, just a way to iterate a NodeList, and back to the comment, I thought that leaving comments is a good idea for writing and learning but I changed my mind and forgot about that one 😅

Reply:
> > * `map` doesn't mutate an array, so you don't need to create a copy
> > * You can prepare one `map`
> > * Don't leave comments, especially in other language except for English, and this comment looks like ChatGPT leaves it
> 
> that map is not for a reason of creating copy, just a way to iterate a NodeList, and back to the comment, I thought that leaving comments is a good idea for writing and learning but I changed my mind and forgot about that one 😅

I mean you copy an array with a spread operator, it doesn't make sense as `map` doesn't mutate an array